source/_sass/lib/site.sass

```
@@ -373,6 +373,35 @@ div.case-studies
       float: right
       margin: 0 0 2ex 3ex
 
+.releases
+  h1
+    font-family: 'Open Sans', sans-serif !important
+
+  h2
+    font-family: 'Open Sans', sans-serif !important
+
+  h3
+    font-family: 'Open Sans', sans-serif !important
+
+  h4
+    font-family: 'Open Sans', sans-serif !important
+
+  h5
+    font-family: 'Open Sans', sans-serif !important
+
+  h6
+    font-family: 'Open Sans', sans-serif !important
+
+  li
+    font-family: 'Open Sans', sans-serif !important
+
+  a
+    font-family: 'Open Sans', sans-serif !important
+
+  p
+    font-family: 'Open Sans', sans-serif !important
```

You don't need to repeat each tag, you can do instead:
```suggestion
.releases
  h1, h2, h3…
    font-family: 'Open Sans', sans-serif !important
```

---

done